docs/SuperSIDonRaspi.md

```diff
@@ -25,60 +43,87 @@ Get the source from GitHub.com
     $ git clone https://github.com/sberl/supersid.git
 ```
 
-To update (pull) to the latest version, do:
-```console
-    ~ $ cd supersid
-    ~/supersid $ git pull
-```
 Now do the following:
 ```console
     ~/supersid $ mkdir Data
     ~/supersid $ mkdir outgoing
 ```
-These directories will be used to store the data that will be sent via ftp to Stanford.
-If your RPi is connected to the internet you can ignore 2. Extra Software and 3.1 optional virtual environment.
-Proceed with the commands under 3.2
+These directories will be used to store the data that will be sent via ftp to
+Stanford.
+
+To update (pull) to the latest version, do:
+```console
+    $ cd ~/supersid
+    $ git pull
+```
+
 
 ## 2) Extra software
 
 Time synchro over the Internet:
+It is important that your system time of day clock is closely synchronized with
+the actual UTC time. Then the data you collect can be compared to data from
+other sources such as other SuperSID users, and Xray flux data collected by
+satellites.
+If your machine does not have ntp installed then you want to install it:
```

Comment:
If you install bullseye as described above, ntp is already configured and enabled automatically.

Reply:
Changeing the documentation from 'buster' to 'bullseye' seems to be a different task. None of  #33, #31, #27, #30, #3, #20 touches the update of the version of the Raspbian OS.

Reply:
> If your machine does not have ntp installed then you want to install it:
This may prevent installing ntp when it is pre-installed.

SuperSIDonRaspbi.md also refers to a configuration guide, which is relevant as well. Is ntp already configured for your country in 'bullseye' after the installation?

Reply:
I followed the steps in SIDonRaspi.md a few days back and found that 1) It installed bullseye by default and 2) Internet time synchronization was already configured and enabled. So the text in the Raspi document should reflect that.

Reply:
Ok, I'll add a clarification to SuperSIDonRaspi.md:

Preparation:
> At the date of the installation (2021-11-01) this corresponds to *buster*.
> Current version as of 2022-01-03 is *bullseye*.

Chapter 2:
> If you installed *Raspbian bullseye*, ntp is already installed and configured.